Jenkinsfile

```diff
@@ -16,9 +16,13 @@ pipeline {
         disableConcurrentBuilds()
         timestamps()
     }
+    triggers {
+        cron(env.BRANCH_NAME == "master" ? "H H(21-23) * * *" : "")
+    }
     environment {
         GITHUB_USERNAME = 'connexta'
         GITHUB_REPONAME = 'multi-int-store'
+        COVERAGE_EXCLUSIONS = '**/test/**/*,**/itests/**/*,**/*Test*,**/sdk/**/*,**/*.js,**/node_modules/**/*,**/jaxb/**/*,**/wsdl/**/*,**/nces/sws/**/*,**/*.adoc,**/*.txt,**/*.xml'
```

Comment:
Why do we need this?
does SonarCloud do code coverage?

Reply:
So that we only analyze classes and not things like xml files. This was copy/pasted from replications repo, so we don't need most of them now.